### DISCUSSION TITLE
Version issue when installing Handbrake container

### DISCUSSION THREAD

**Discussion by gsq1016** [jlesage/docker-handbrake#273] [Q&A]
I was wondering why the latest version I installed using the 'docker' command was 1.4.2, while others have version 1.6.1. I'm not sure if I did something wrong during the installation process.

**Comment by jlesage** [comment]
What did you do ?  Maybe you have an old image, make to download the latest one with `docker pull jlesage/handbrake` before creating the container.

**Reply by gsq1016** [reply]
It's been resolved. It was a network issue, so I used another Wi-Fi to download the latest version. Even though the download was slow.

**Comment by jlesage** [comment]
Ok, great to know that you were able to solve your problem.
